src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -105,49 +115,85 @@ const DateTimePicker = (props: Props) => {
     todayButton,
     startDate,
     withPortal,
+    isValid,
+    isInvalid,
+    invalidFeedback,
+    validFeedback,
   } = props
 
   return (
-    <InputGroup className={className}>
-      <InputGroup.Prepend>
-        <InputGroup.Text>
-          <Icon icon="calendar" />
-        </InputGroup.Text>
-      </InputGroup.Prepend>
-      <DatePicker
-        className="form-control"
-        dateFormat={dateFormat}
-        dateFormatCalendar={dateFormatCalendar}
-        disabled={disabled}
-        dropdownMode={dropdownMode}
-        endDate={endDate}
-        excludeDates={excludeDates}
-        includeDates={includeDates}
-        inline={inline}
-        locale={locale}
-        maxDate={maxDate}
-        maxTime={maxTime}
-        minDate={minDate}
-        minTime={minTime}
-        monthsShown={monthsShown}
-        onChange={onChange}
-        selected={selected}
-        selectsEnd={selectsEnd}
-        selectsStart={selectsStart}
-        showMonthDropdown={showMonthDropdown}
-        showTimeSelect={showTimeSelect}
-        showYearDropdown={showYearDropdown}
-        showTimeSelectOnly={showTimeSelectOnly}
-        timeFormat={timeFormat}
-        timeIntervals={timeIntervals}
-        timeCaption={timeCaption}
-        todayButton={todayButton}
-        startDate={startDate}
-        withPortal={withPortal}
-      >
-        {children}
-      </DatePicker>
-    </InputGroup>
+    <div>
```

Comment:
rather than rendering a div, we can just render a `<>` and close it with `</>` so we don't render extra dom elements.